### PR TITLE
Removed EBI layer on PWA-related pipelines

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
Ticket [ENSCOMPARASW-2486](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-2486) (part 2/7).

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm [hunk 1]
```diff
@@ -21,9 +21,23 @@ limitations under the License.
...
+    [Alternative 2] Provide the collection and the reference species:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Lastz_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -division $COMPARA_DIV -collection collection-e94_new_species_human_lastz -ref_species homo_sapiens
+
 =head1 DESCRIPTION  
 
-This is a base configuration files for all LastZ runs.
+This is a base configuration file for LastZ pipeline.
 It is not intended to be run on its own, it just turns the generic
 PairAligner pipeline into a LastZ one.
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm]: The instructions above make it seem like it's fine to run on its own, so maybe remove this?

File: modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm [hunk 2]
```diff
@@ -25,26 +25,22 @@ Bio::EnsEMBL::Compara::PipeConfig::TBlat_conf
...
     #4. Check all default_options below, especially
         ref_species (if not homo_sapiens)
         default_chunks
         pair_aligner_options
 
     #5. Run init_pipeline.pl script:
-        Using command line arguments:
-        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::TBlat_conf --dbname drer_onil_tblat_67 --password <your password> --mlss_id 574 --pipeline_db -host=compara1 --ref_species danio_rerio --pipeline_name TBLAT_dr_on_67
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::TBlat_conf -host mysql-ens-compara-prod-X -port XXXX \
+            --mlss_id 574
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm]: Add `-division vertebrates -ref_species danio_rerio`

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/TBlat_conf.pm [hunk 3]
```diff
@@ -19,11 +19,17 @@ limitations under the License.
...
 =head1 NAME
 
-Bio::EnsEMBL::Compara::PipeConfig::EBI::TBlat_conf
+Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::TBlat_conf
+
+=head1 SYNOPSIS
+
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::TBlat_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -mlss_id_list "[9769,9784,9754,9770,9785,9786,9771,9755,9767,9768,9782,9783,9756,9752,9753]" \
+        -pipeline_name ${COMPARA_DIV}_tblat_batchX_${CURR_ENSEMBL_RELEASE}
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/TBlat_conf.pm]: Use instead `-mlss_id 591 -ref_species mus_musculus`. Or `-mlss_id 574 -ref_species danio_rerio` like in the parent class

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm [hunk 4]
```diff
@@ -49,26 +63,38 @@ sub default_options {
     return {
 	    %{$self->SUPER::default_options},   # inherit the generic ones
 
-	    #
+        'master_db' => 'compara_master',
+
+        # Work directory
+        'dump_dir' => $self->o('pipeline_dir'),
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm]: This is the same as in TBlat_conf. Move to PairAligner_conf

File: modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm [hunk 5]
```diff
@@ -69,6 +65,17 @@ sub default_options {
     return {
 	    %{$self->SUPER::default_options},   # inherit the generic ones
 
+        # We connect to the databases via the Registry configuration file of the division
+        'master_db'              => 'compara_master',
+        'curr_core_sources_locs' => undef,
+        'curr_core_dbs_locs'     => undef,
+
+        # Work directory
+        'dump_dir' => $self->o('pipeline_dir'),
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm]: This is the same as in LastZ_conf. Move to PairAligner_conf

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm [hunk 6]
```diff
@@ -49,26 +63,38 @@ sub default_options {
     return {
 	    %{$self->SUPER::default_options},   # inherit the generic ones
 
-	    #
+        'master_db' => 'compara_master',
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Lastz_conf.pm]: This is our standard way of running pipelines. Move to PairAligner_conf

File: modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm [hunk 7]
```diff
@@ -69,6 +65,17 @@ sub default_options {
     return {
 	    %{$self->SUPER::default_options},   # inherit the generic ones
 
+        # We connect to the databases via the Registry configuration file of the division
+        'master_db'              => 'compara_master',
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm]: This is our standard way of running pipelines. Move to PairAligner_conf

File: modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm [hunk 8]
```diff
@@ -69,6 +65,17 @@ sub default_options {
...
+        'curr_core_sources_locs' => undef,
+        'curr_core_dbs_locs'     => undef,
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm]: These two parameters are not passed to the `parse_pair_aligner_conf` analysis any more. Remove all occurrences in all PWA PipeConfigs (except the Legacy ones)

**Comment by JAlvarezJarreta** [comment]
Travis is failing because the newly added `PipeConfig::EBI::Metazoa::Lastz_conf` fails to compile as the base class (`PipeConfig::EBI::Lastz_conf`) doesn't exist anymore. This has been addressed in another [PR](https://github.com/Ensembl/ensembl-compara/pull/154).

**Pull request merged**